bigquery_etl/routine/publish_routines.py

```diff
@@ -124,9 +124,10 @@ def publish_routine(
         dataset = client.create_dataset(raw_routine.dataset, exists_ok=True)
 
         # set permissions for dataset, public for everyone
-        entry = bigquery.AccessEntry("READER", "specialGroup", "allAuthenticatedUsers")
+        read_entry = bigquery.AccessEntry("READER", bigquery.enums.EntityTypes.SPECIAL_GROUP, "allAuthenticatedUsers")
+        write_entry = bigquery.AccessEntry("WRITER", bigquery.enums.EntityTypes.IAM_MEMBER, "workgroup:platform-infra/internal")
```

Comment:
I was unable to figure out what `workgroup:platform-infra/internal` actually is in [this list](https://googleapis.dev/python/bigquery/latest/enums.html#google.cloud.bigquery.enums.EntityTypes), so I went with `IAM_MEMBER`

Reply:
You need to use `group:team-data-platform@firefox.gcp.mozilla.com`, workgroups aren't a GCP-level IAM concept

Reply:
Is the EntityType `IAM_MEMBER` correct?